coredns/go.mod

```diff
@@ -0,0 +1,29 @@
+module github.com/submariner-io/lighthouse/coredns
+
+go 1.13
+
+require (
+	github.com/caddyserver/caddy v1.0.5
+	github.com/coredns/caddy v1.1.1
+	github.com/coredns/coredns v1.9.1
+	github.com/miekg/dns v1.1.48
+	github.com/onsi/ginkgo v1.16.5
+	github.com/onsi/gomega v1.19.0
+	github.com/pkg/errors v0.9.1
+	github.com/prometheus/client_golang v1.12.1
+	github.com/submariner-io/lighthouse v0.12.0-m3.0.20220502131803-864fecb47b2f
```

Comment:
What's the implication here?

We will have to update it during release?

Also will LH changes pertaining to both now have to be done in 2 steps?

Local testing of changes is now more problematic?

Reply:
I’ve added a local replace so we don’t need to worry about any of this in practice.